scmail.py

```diff
@@ -134,42 +143,60 @@ def register(ctx, fingerprint):
 
 
 @click.command()
-@click.option('--fingerprint', '-f', prompt='Enter fingerprint of mailbox', required=True, help='The fingerprint of the yourself mailbox.')
-@click.option('--sender-fingerprint', '-s', prompt='Enter the fingerprint of sender', default='', help='The senders fingerprint.')
-@click.password_option('--password', '-p', prompt='Enter password of private key', help='The passphrase of private key')
+@click.option('--fingerprint', '-f', prompt='Enter fingerprint of mailbox',
+              required=True, type=str, help='The fingerprint of the yourself mailbox.')
+@click.option('--sender-fingerprint', '-s', prompt='Enter the fingerprint of sender',
+              type=str, help='The senders fingerprint.')
```

Comment:
Set default parameter to make `sender_fingerprint` optional when retrieving:
```suggestion
              default='', type=str, help='The senders fingerprint.')
```